docs/src/SubNav.js

```diff
@@ -53,9 +53,16 @@ class SubNav extends React.Component {
     }
 
     if (props.children) {
-      return ValidComponentChildren.some(props.children, child => (
-        this.isActive(child, activeKey, activeHref)
-      ));
+      let active;
+      ValidComponentChildren.forEach(props.children, (child) => {
+        if (active === undefined) {
+          active = this.isActive(child, activeKey, activeHref);
+        }
+      });
+
+      if (active !== undefined) {
```

Comment:
you can use to check if any child is active is so return true otherwise undefined? seems like all you need is the if block guarding against returning false right?